packages/hint-manifest-icons/src/_locales/en/messages.json

```diff
@@ -11,6 +11,14 @@
         "description": "Report message when an icon request return an status code !== 200",
         "message": "Icon could not be fetched (status code: $1)."
     },
+    "iconPurposeDuplicate": {
+        "description": "Report message when an icon has a duplicate value in the property purpose",
+        "message": "Duplicate value(s) found in icon purpose porperty: $1."
```

Comment:
missed misspelling
```suggestion
        "message": "Duplicate value(s) found in icon purpose property: $1."
```

Reply:
🤦‍♂️ I forgot to save my changes before pushing...
![image](https://user-images.githubusercontent.com/1581288/86068080-4576e000-ba2b-11ea-9e04-d31527d5a500.png)